packages/moonstone/styles/colors-light.less

```diff
@@ -101,7 +101,7 @@
 
 // FormCheckbox
 // ---------------------------------------
-@moon-formcheckbox-bg-color: #686868;
+@moon-formcheckbox-bg-color: @moon-white;
```

Comment:
This change causes the checkbox to not be visible when unfocused in light skin because they are the same color. Not sure the background of this change but we either need to make the text color or the bg color different.